src/legacy/server/status/routes/api/register_stats.js

```diff
@@ -88,7 +88,7 @@ export function registerStatsApi(kbnServer, server, config) {
             ]);
 
             let modifiedUsage = usage;
-            if (isLegacy) {
+            if (isLegacy && modifiedUsage) {
```

Comment:
I'm not sure this is the right fix. If `modifiedUsage` is falsy, we'd still want https://github.com/elastic/kibana/pull/36986/files#diff-f0c0c68b870440940fc9df81085b5249R123 to be executed, right?

Instead, perhaps we could make https://github.com/elastic/kibana/pull/36986/files#diff-f0c0c68b870440940fc9df81085b5249R83 return `Promise.resolve({})` instead of `Promise.resolve()`. WDYT?

Reply:
You're right. That is better. I'll update it